README.md

```diff
@@ -441,14 +460,18 @@ For an example implementation view the [connect-redis](http://github.com/visionm
 This optional method is used to get all sessions in the store as an array. The
 `callback` should be called as `callback(error, sessions)`.
 
-### store.destroy(sid, callback)
+### store.destroy(sid, [req], callback)
```

Comment:
I think it might be good to use an object to pass the request to the store to make future changes easier if something else needs to be passed to the store.